lib/matplotlib/scale.pyi

```diff
@@ -56,12 +57,12 @@ class LogScale(ScaleBase):
     name: str
     subs: Iterable[int] | None
     def __init__(
-        self,
-        axis: Axis | None,
+        self: LogScale,
+        axis: Union[Axis, None] = None,
```

Comment:
```suggestion
        axis: Axis | None = ...,
```

`Axis | None` is equivalent to `Union[Axis, None]`, but the preferred spelling. You don't repeat default values in stub files. Instead, you place an ellipsis.